scripts/release_utils/consortium_to_bpc.py

```diff
@@ -63,16 +63,126 @@ def find_release(
         return releasedf.iloc[0, 0]
 
 
-def remove_gene_panels(syn, file_mapping, remove_seqassays, remove_centers):
-    """ NOTE: Is this still needed?
-        Removes gene panels that shouldn't be there
+def _copyRecursive(
+    syn: synapseclient.Synapse,
+    entity: str,
+    destinationId: str,
+    mapping: Dict[str, str] = None,
+    skipCopyAnnotations: bool = False,
+    **kwargs,
+) -> Dict[str, str]:
     """
-    for name in file_mapping:
-        gene_name = name.replace("data_gene_panel_", "").replace(".txt", "")
-        if gene_name in remove_seqassays or gene_name.startswith(tuple(remove_centers)):
-            print(name)
-            print(file_mapping[name])
-            syn.delete(file_mapping[name])
+    NOTE: This is a copy of the function found here: https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/synapseutils/copy_functions.py#L409
```

Comment:
Is this for copying any entities with AR (even within same project just to different folder, just any usage of `synapseutils.copy`)?

Reply:
In the client the `synu.copy` command will not work on any entities that have access requirements.

Reply:
Can you link to that code? 😅

Reply:
https://github.com/Sage-Bionetworks/Genie/blob/90d54c54139a356423681f4bba5aec2b3cdc788c/genie/consortium_to_public.py#L341

Wondering if the copy function could have a parameter to override the AR?

Reply:
@rxu17 thats a good question - we cannot have a parameter to override the AR because it's meant to be inconvenient for people. 

That said, there's nothing stopping anyone from downloading and uploading data